lib/config-loader.js

```diff
@@ -14,7 +14,9 @@ const loadConfig = function (path) {
         data = data.slice(1);
     }
 
-    return JSON.parse(data);
+    data = JSON.parse(data);
+    data._path = path;
```

Comment:
I'm personally not a fan of properties led with an underscore. I think `data.filePath` or `data.configPath` would be good and semantic here